core/src/main/scala/zio/config/Config.scala

```diff
@@ -33,21 +33,21 @@ object Config {
   def fromEnv[K, V, A](
     configDescriptor: ConfigDescriptor[String, String, A],
     valueDelimiter: Option[String] = None
-  ): IO[ReadErrors[Vector[String], String], Config[A]] =
+  ): IO[ReadErrors[Vector[String]], Config[A]] =
     make(ConfigSource.fromEnv(valueDelimiter), configDescriptor)
 
   def fromMap[A](
     map: Map[String, String],
     configDescriptor: ConfigDescriptor[String, String, A],
     pathDelimiter: String = "."
-  ): IO[ReadErrors[Vector[String], String], Config[A]] =
+  ): IO[ReadErrors[Vector[String]], Config[A]] =
```

Comment:
ReadErrors knew about `V` only for message purpose. I don't think its worth it considering the complexity that it caused across the entire applications.

ReadErrors is associated with a key.  The only time value is used is to print out a message of parse errors